flocker/acceptance/testtools.py

```diff
@@ -109,14 +111,20 @@ def create_attached_volume(dataset_id, mountpoint, maximum_size=None,
     )
 
 
-def get_node_state(node):
+def get_node_state(hostname):
```

Comment:
Maybe this makes sense as a method of `Cluster` now?  The call to `get_test_cluster()` makes me unhappy.  I see how magically it can return a new `Cluster` instance that's usable in this context - for now, at least.  Relying on that method to be idempotent seems like it greatly constraints future flexibility, though.